lib/flowViewport/sceneIndex/fvpPathInterface.h

```diff
@@ -32,14 +32,30 @@ class Path;
 
 namespace FVP_NS_DEF {
 
+// Based on USD's HdInstanceIndicesSchema : 
+// https://github.com/PixarAnimationStudios/OpenUSD/blob/59992d2178afcebd89273759f2bddfe730e59aa8/pxr/imaging/hd/instanceIndicesSchema.h
+struct InstancesSelection {
```

Comment:
New struct to better match Hydra's data structures, and which allows us to specify the instancer and prototype paths, for native instances prototypes selection.